spanner/spanner_leaderboard/leaderboard.go

```diff
@@ -94,15 +114,16 @@ func insertPlayers(ctx context.Context, w io.Writer, client *spanner.Client) err
 		return err
 	}
 	// Intialize values for random PlayerId
-	rand.Seed(time.Now().UnixNano())
-	min := 1000000000
-	max := 9000000000
+	min := int64(1000000000)
+	max := int64(9000000000)
+	rnd := newUniqueRand()
 	// Insert 100 player records into the Players table
 	_, err = client.ReadWriteTransaction(ctx, func(ctx context.Context, txn *spanner.ReadWriteTransaction) error {
 		stmts := []spanner.Statement{}
 		for i := 1; i <= 100; i++ {
 			numberOfPlayers++
-			playerID := rand.Intn(max-min) + min
+			playerID := rnd.rnd(min, max)
```

Comment:
Does the player ID need to be an int? Can we use a UUID instead?

Reply:
I don't think we should change that. This sample is used across all supported programming languages and uses the same data model for all of them.